package.json

```diff
@@ -61,6 +61,7 @@
     "postversion": "yarn generate --force && yarn doc && ts-node scripts/after-bump.ts",
     "lint": "turbo run lint",
     "lint:fix": "turbo run lint:fix --force",
+    "checks": "turbo run check && yarn run check:circular && yarn run check:license && yarn run check:test-service",
```

Comment:
Some of the checks use turbo, and some of them don't (that's the status quo).

<img width="431" alt="Screenshot 2023-03-23 at 10 54 23" src="https://user-images.githubusercontent.com/2292245/227167085-337f1778-5fda-46ab-8374-892cf4f647a7.png">

My understanding from what @florian-richter explained about the reasoning is that turbo is used for steps which have scripts in packages and thus can benefit from being run in parallel, compared to steps which only operate on the whole project where turbo would not benefit us.

I'm not sure if it has anything todo with 'needs compile' or not.

My first impulse was to put all checks into turbo, but then I wanted to keep the change relatively small and I did not see the large benefit, that's why I've ended up with this multi-command for what "checks" does.

Reply:
Ok, but then still: Why does the `turbo run check` step (semantically) **depend on** the dependency check. The `turbo run check` should (semantically) **execute** the dependency check. Technically it will be executed, but the configuration seems unintended to me.

Reply:
This might be my missing knowledge about how to configure turbo, but is there another way to express this? Looking at the [docs](https://turbo.build/repo/docs/reference/configuration) I don't see it, and we don't seem to do anything else in any of our tasks in turbo json..